opentelemetry/benches/metrics.rs

```diff
@@ -1,4 +1,19 @@
-use criterion::{criterion_group, criterion_main, Criterion};
+/*
+    Benchmark Results:
+    criterion = "0.5.1"
+    OS: Ubuntu 22.04.4 LTS (5.15.153.1-microsoft-standard-WSL2)
+    Hardware: Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz, 16vCPUs,
+    RAM: 64.0 GB
+    | Test                                                | Average time|
+    |-----------------------------------------------------|-------------|
+    | NoAttributes                                        | 1.1616 ns   |
+    | AddWithInlineStaticAttributes                       | 13.296 ns   |
+    | AddWithStaticArray                                  | 1.1612 ns   |
+    | AddWithDynamicAttributes                            | 110.40 ns   |
```

Comment:
This benchmark method runs much faster when run in isolation. It takes around 80 ns instead of 110 ns.